tests/test_landing_job.py

```diff
@@ -90,7 +90,9 @@ def test_cancel_landing_job_fails_bad_input(db, client, landing_job, auth0_mock)
     )
 
     assert response.status_code == 400
-    assert response.json["detail"] == ("'IN_PROGRESS' is not one of ['CANCELLED']")
+    assert response.json["detail"] == (
```

Comment:
Is the addition of ` - 'status'` leftover from when this PR had Flask v2? Confused as to how this changes after this PR.

Reply:
This is because of the upgrade to connexion, not Flask.